### DISCUSSION TITLE
Problems with Staticman on Heroku

### DISCUSSION THREAD

**Discussion by pmatulis** [eduardoboucas/staticman#430] [Q&A]
Hi,

I've looked at quite a few documentation guides and blogs but I can't get Staticman to work. I have it running on an Heroku instance and [appears to be running](https://staticman-pmatulis.herokuapp.com/). I'm having it connect to an intermediary GitHub account and have generated an auth token. In Heroku I have input config variables `GITHUB_TOKEN` and `RSA_PRIVATE_KEY`.  However, when I try to test the workflow with [this blog post](https://ubuntustack.com/compute/lithium/) from a Jekyll-based [site](https://github.com/pmatulis/ubuntustack) it fails. In the Heroku app [logs](https://paste.ubuntu.com/p/HNGxQfd7kd/) it shows ```Error: Require an `oauthToken` or `token` option```. What am I missing? Should I have to input another variable in Heroku for specifying the intermediary GitHub account like `GITHUB_USRENAME`?

Thanks in advance for any input you may have.

**Comment by VincentTam** [comment]
Hi I've set up the world's first GitLab Pages-powered Staticman blog four years ago.  [My outdated guides ](https://vincenttam.gitlab.io/post/2018-09-16-staticman-powered-gitlab-pages/1/)have been referred to for a number of times.
the linked blog post shows 404 error.
